docs/pages/posts/2022/8/27/secureum-a-maze-x-stanford-ctf.mdx

```diff
@@ -426,12 +432,11 @@ This challenge was a lesson on why to never trust DEX's current prices as a pric
 ---
 
 import { Card, Cards } from 'nextra/components'
-import { FileIcon, NewsletterIcon } from '@components/icons'
+import { NewsletterIcon } from '@components/icons'
 
-<Cards>
```

Comment:
Can't possibly compile without this.

Again, please check your changes properly before submitting them.